### PR TITLE
Add new menu user options

### PR DESCRIPTION
**Pull request by adudekyeti**
**Desktop:**

image

---
**Mobile:**

image

### CODE REVIEW

File: app/Controller/View/Page.php [hunk 1]
```diff
@@ -231,6 +213,64 @@ protected function getMenuHeaderLinks(\App\Request $request)
 		return $headerLinkInstances;
 	}
 
+	/**
+	 * Function to get the list of User Header Links.
```
Comment (mariuszkrzaczkowski) [inline app/Controller/View/Page.php]: ?

File: layouts/basic/modules/Vtiger/UserQuickMenu.tpl [hunk 2]
```diff
@@ -0,0 +1,63 @@
+{*<!-- {[The file is published on the basis of YetiForce Public License 3.0 that can be found in the following directory: licenses/LicenseEN.txt or yetiforce.com]} -->*}
+{strip}
```
Comment (mariuszkrzaczkowski) [inline layouts/basic/modules/Vtiger/UserQuickMenu.tpl:2]:
```
{strip}
<!-- tpl-Base-UserQuickMenu -->
<div .......
```

File: layouts/basic/modules/Vtiger/UserQuickMenu.tpl [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+									<span>{\App\Language::translate($TITLE, 'Users')}</span>
+								{/if}
+							</a>
+						</div>
+					</div>
+				{/if}
+			{/foreach}
+		</div>
+	</div>
+{/strip}
```
Comment (mariuszkrzaczkowski) [inline layouts/basic/modules/Vtiger/UserQuickMenu.tpl:65]:
```
	</div>
<!-- /tpl-Base-UserQuickMenu -->
{/strip}
```

**Review by mariuszkrzaczkowski — CHANGES_REQUESTED** [review]

**Comment by rskrzypczak** [comment]
try to move everything about the user to the Users module

**Pull request merged**